ios/Codeforces Watcher/Application/features/news/cells/FeebackView.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftUI
+
+struct FeedbackView: View {
+    
+    var post: NewsItem.FeedbackItem
+    var callback: () -> Void = {}
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            HStack(alignment: .top) {
+                TitleView
```

Comment:
I didn't know we can build SwiftUI views with `vars`, but it looks a bit weird (not having `()`). Is it a common practice?

Reply:
Also, here, I think that empty lines decrease readability.

Reply:
Agree with you about the parentheses, but yes, that's common practice too(body is a `var`), I use it as a `var` if I don't need to pass parameters to the subview and use a function otherwise.

Reply:
@derevyanko OK, let's leave it as it is, and then return too it after your Android experience with Jetpack Compose 😉